pkgs/by-name/bi/bigquery-emulator/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  buildGoModule,
+  pkgs,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  version = "0.6.6";
+  pname = "bigquery-emulator";
+in
+buildGoModule.override
+  {
+    stdenv = pkgs.clangStdenv;
+  }
+  {
+    name = pname;
+    src = fetchFromGitHub {
+      owner = "goccy";
+      repo = pname;
```

Comment:
pname should not be used like this

Reply:
https://github.com/NixOS/nixpkgs/pull/367622